src/backends/plotly.jl

```diff
@@ -448,9 +454,15 @@ const _plotly_markers = KW(
     :hline      => "line-ew",
 )
 
-function plotly_subplot_index(sp::Subplot)
-    spidx = sp[:subplot_index]
-    spidx == 1 ? "" : spidx
+# find indicies of axes to which the supblot links to
+function plotly_link_indicies(plt::Plot, sp::Subplot)
```

Comment:
Is there a way to access the `Plot` from a `Subplot`, I couldn't find anything 😟

Reply:
Does `sp.plt` work in this context?

Reply:
Yeah that's what I was looking for, not sure how I missed that possibility 😆

Reply:
Although now that  think about it it seems clearer to have the `Plot` in the function calls if it's attributes are going to be used. So I think it's fine as it is now 😄